src/components/Global/Navbar.vue

```diff
@@ -3,36 +3,95 @@
     class="
       text-black
       dark:text-white
-      sm:flex sm:justify-between sm:items-center sm:px-4 sm:py-3
+      burger:flex burger:justify-between burger:items-center burger:px-4 burger:py-3
       border-b-2
       dark:border-b-0 dark:shadow-xl
     "
   >
     <div
       class="
         flex flex-wrap
-        sm:flex-nowrap
+        burger:flex-nowrap
         w-full
         items-center
         justify-between
         align-baseline
         px-4
         py-3
-        sm:p-0
+        burger:p-0
       "
     >
       <!-- Dev Protocol Icon -->
-      <a href="https://devprotocol.xyz" class="text-gray-900">
-        <img src="/assets/logo.png" alt="DevProtocol" class="max-h-8" />
+      <a href="https://devprotocol.xyz" class="text-gray-900 p-4">
+        <svg
```

Comment:
Maybe we can add these SVGs to a file and load them here? It'll make the code shorter, IMO.

Reply:
it edits the SVG's internals to work with dark mode. 🙂